pkg/backend/vxlan/vxlan.go

```diff
@@ -144,21 +144,22 @@ func (be *VXLANBackend) RegisterNetwork(ctx context.Context, wg *sync.WaitGroup,
 	var err error
 
 	// When flannel is restarted, it will get the MAC address from the node annotations to set flannel.1 MAC address
-	var hwAddr net.HardwareAddr
+	var hwAddr, hwAddrv6 net.HardwareAddr
 
-	macStr := be.subnetMgr.GetStoredMacAddress(ctx)
+	macStr, macStrv6 := be.subnetMgr.GetStoredMacAddresses(ctx)
+	log.Infof("MANU - This is macStr and this is macStrv6: %s %s", macStr, macStrv6)
```

Comment:
MANU?

Reply:
haha that's my log, sorry it's not 100% ready yet because I found some extra stuff